File: eDRS Land Registry/eDRS Land Registry/Models/ApplicationForm.cs
using System;

namespace eDRS_Land_Registry.Models
{
    public class ApplicationForm
    {
        public long ApplicationFormId { get; set; }
        public string Variety { get; set; }
        public int Priority { get; set; }
        public string Value { get; set; }
        public int FeeInPence { get; set; }
        public string Type { get; set; }
        public string ExternalReference { get; set; }
        public string CertifiedCopy { get; set; }
        public DateTime ChargeDate { get; set; }
        public string IsMdRef { get; set; }
        public string SortCode { get; set; }
        public string MdRef { get; set; }
        public bool IsChecked { get; set; }
        public long DocumentReferenceId { get; set; }
        public Document Document { get; set; }

    }

    public class Document
    {

        public long DocumentId { get; set; }
        public long AttachmentId { get; set; }

        public string Base64 { get; set; }
        public string FileName { get; set; }
        public string FileExtension { get; set; }

        public long ApplicationFormId { get; set; }

    }
}

File: FrontEnd/src/app/models/application-form.ts
export class ApplicationForm {
    ApplicationFormId?: number;
    Priority?: number;
    Value?: string;
    FeeInPence?: number;
    Type?: string;
    LocalId?: any;
    IsSelected?: boolean;
    Document?: Document;
    Variety?: string;
    IsChecked?: boolean;

}

export class Document {
    DocumentId?: number;
    CertifiedCopy?: string;
    Base64?: string;
    FileName?: string;
    FileExtension?: string;
    ApplicationFormId?: number;

}
